crmsh/bootstrap.py

```diff
@@ -1777,6 +1781,52 @@ def join_ssh(seed_host):
         error("Can't invoke crm cluster init -i {} ssh_remote on {}".format(_context.default_nic_list[0], seed_host))
 
 
+def swap_public_ssh_key(remote_node):
+    """
+    Swap public ssh key between remote_node and local
+    """
+    # Detect whether need password to login to remote_node
+    if utils.check_ssh_passwd_need(remote_node):
+        # If no passwordless configured, paste /root/.ssh/id_rsa.pub to remote_node's /root/.ssh/authorized_keys
+        status("Configuring SSH passwordless with root@{}".format(remote_node))
+        # This time login to remote_node is passwordless
+        append_to_remote_file(RSA_PUBLIC_KEY, remote_node, AUTHORIZED_KEYS_FILE)
+
+    try:
+        # Fetch public key file from remote_node
+        public_key_file_remote = fetch_public_key_from_remote_node(remote_node)
+    except ValueError as err:
+        warn(err)
+        return
+    else:
```

Comment:
Is here some particular reason why you are using this `else` statement? It looks pretty strange.
The unique reason why I would add it is that `append_unique` can raise a `ValueError` and you don't want to catch it for some reason. Otherwise you could just put it in the `try` block.
Just curiosity hehe

Reply:
@arbulu89 I've removed the `else` keyword and move forward the codes under `else`